app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    without_mask = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
```

Comment:
change order for except because VaccineError have bigger priority in this task

Reply:
OK, I`ll do that, but if VaccineError is occure, it will be raised first in visit_cafe. It is only handles here.